infrastructure/main.tf

```diff
@@ -55,6 +63,12 @@ module "feature-toggle-api" {
     FLYWAY_URL                  = "jdbc:postgresql://${module.feature-toggle-db.host_name}:${module.feature-toggle-db.postgresql_listen_port}/${module.feature-toggle-db.postgresql_database}${local.db_connection_options}"
     FLYWAY_USER                 = "${module.feature-toggle-db.user_name}"
     FLYWAY_PASSWORD             = "${module.feature-toggle-db.postgresql_password}"
+    TEST_ADMIN_USERNAME         = "${local.test_admin_user}"
+    TEST_ADMIN_PASSWORD         = "${local.test_admin_password}"
+    TEST_EDITOR_USERNAME        = "${local.test_editor_user}"
+    TEST_EDITOR_PASSWORD        = "${local.test_editor_password}"
+    TEST_READ_USERNAME          = "${local.test_read_user}"
+    TEST_READ_PASSWORD          = "${local.test_read_password}"
```

Comment:
are those env vars for functional tests only? If so, are those needed?

Reply:
PR is closed so it would be good if we review it at once or if you have issues slack or face to face would be better.
Yes those are for functional tests only.Yes those are needed.